tpp-app/src/home/pages/PrivacyPolicyScreen.js

```diff
@@ -1,14 +1,26 @@
 import React from 'react';
-import { View, ScrollView, StyleSheet, Text, useWindowDimensions, TouchableOpacity, ImageBackground, SafeAreaView} from 'react-native';
-import RenderHtml from 'react-native-render-html';
+import { View, ScrollView, StyleSheet, Text, useWindowDimensions, TouchableOpacity, ImageBackground, Linking} from 'react-native';
 import BackIcon from '../../../ios/tppapp/Images.xcassets/icons/back_icon.svg'
 import OnboardingBackground from '../../../ios/tppapp/Images.xcassets/SplashScreenBackground.imageset/colourwatercolour.png'
 
+const EmailLink = () => {
+    const openLink = () => {
+        
+            Linking.canOpenURL("mailto:hello@periodpurse.com").then(() => {
```

Comment:
The mail to link doesn't seem to work

Reply:
i get this as well but roy you tested on faye's phone and it works alright there?

Reply:
works on phone :)